Imagetospeech/image_to_speech.py

```diff
@@ -0,0 +1,30 @@
+import pytesseract
+from PIL import Image
+from gtts import gTTS
+import os
+
+
+pytesseract.pytesseract.tesseract_cmd = input(r'Enter the path for pytesseract: ')
+
+img = input(r"Enter image path: ")
+target = Image.open(img)
+text = pytesseract.image_to_string(target, config='')
+
+with open(f"./Imagetospeech/Text/text.txt",'w') as f:
```

Comment:
```suggestion

with open(f"./Imagetospeech/text.txt",'w') as f:
```
@Amit366 as suggested by @kaustubhgupta remove the Text folder since there is no Text folder